files/organizations/allowed_regions.json.tpl

```diff
@@ -28,6 +28,7 @@
                 "cloudwatch:Describe*",
                 "cloudwatch:Get*",
                 "cloudwatch:List*",
+                "logs*",
```

Comment:
please alphabetize and correct the policy logs* is not correct it needs to be logs:*

Reply:
Have updated @marwinbaumannsbp  . Thanks